signer/src/bitcoin/packaging.rs

```diff
@@ -64,9 +64,10 @@ where
     I: IntoIterator<Item = T>,
     T: Weighted,
 {
-    // This is a variant of the Best-Fit-Decreasing algorithm, so we sort
-    // by "weight" decreasing. We use the votes against as the weight, but
-    // vsize is a reasonable weight metric as well.
+    // We might want to sort the items, and previously the was sorted by votes.
+    // However, it can allow malicious signer to speedup/slowdown requests execution,
+    // so it is not so good key to sort with. Some better keys can be time or something
+    // like this, but it require a bit more work so for now we just keep it unsorted.
```

Comment:
```suggestion
    // This was previously sorted by votes, however we now leave it as an
    // unsorted list due to the risk of a malicious signer being able to effectively
    // influence the ordering of request processing.
```

Reply:
Now we removed this comment entirely https://github.com/stacks-sbtc/sbtc/pull/1664#discussion_r2080240130